+ Styled Chat & Styled Player List - Prefixes, Suffixes, & Meta

cswimr opened the pull request:
These two mods are the only good solutions I've found for 1.19 chat/tablist formatting on Fabric, now that GraphiXMod and Chatter aren't actively updated to new versions anymore.

cswimr committed:
+ Styled Chat & Styled Player List

These two mods are the only good solution I've found for 1.19 chat/tablist formatting on Fabric, now that GraphiXMod and Chatter aren't actively updated to new versions anymore.

cswimr committed:
changed Modrinth links to GitHub links

Patbox commented:
I would prefer if you kept modrinth links :p

Also it would be worth noting that manual configuration (and maybe using LuckPerms placeholder addon) is required, as it won't work out of the box

cswimr committed:
changed github links back to modrinth links

Patbox stated he prefers Modrinth links, so I've undone my previously latest commit.

cswimr commented:
Thanks for the comment! I'll add the changes you requested in my next commit.

cswimr committed:
added more information per patbox's comment

cswimr committed:
specified Fabric placeholderapi
